rgw/v2/utils/utils.py

```diff
@@ -554,6 +554,17 @@ def check_bucket_sync(name):
     return out
 
 
+def bucket_sync_status(name):
+    cmd = f"radosgw-admin bucket sync status --bucket={name}"
+    out = exec_shell_cmd(cmd)
+    if "bucket is caught up with source" in out:
+        log.info("bucket sync status is caught up with the other site")
+        return True
+    else:
+        log.info("bucket is few behind shards")
+        return False
+
```

Comment:
Since we are dealing with multisite sync. Can we add delay in form of delay and retries.
Referance: https://github.com/red-hat-storage/ceph-qe-scripts/blob/82a2db9fc5d36d855f2a53cd68546adf95af4045/rgw/v2/lib/sync_status.py#L14

Reply:
done